### PR TITLE
Update authors.json

### PR DESCRIPTION
**Pull request by mjbuys**
Please can someone check this PR as I seem to have disappeared from the author list although I have published previous posts. If I indeed was removed then please check syntax and merge into master branch.

---
This change is [Reviewable](https://reviewable.io/reviews/datacite/segugio/13)

### CODE REVIEW

File: source/data/authors.json [hunk 1]
```diff
@@ -52,6 +52,18 @@
...
+  },
+  "mbuys": {
+    "name": "Matt Buys",
+    "given": "Matt",
+    "family": "Buys",
+    "bio": "DataCite Executive Director",
+    "orcid": "https://orcid.org/0000-0001-7234-3684",
+    "email": "matt.buysr@datacite.org",
+    "image_url": "/images/mbuys.jpg",
+    "affiliation": { "@id": "https://ror.org/04wxnsj81", "name": "DataCite", "@type": "Organization"},
```
Comment (kjgarza) [inline source/data/authors.json:64]: i think affiliation is not part of the hash, maybe we should remove it.

File: source/data/authors.json [hunk 2]
```diff
@@ -52,6 +52,18 @@
...
     "image_url": "/images/rhallett.png"
+  },
+  "mbuys": {
+    "name": "Matt Buys",
+    "given": "Matt",
+    "family": "Buys",
+    "bio": "DataCite Executive Director",
+    "orcid": "https://orcid.org/0000-0001-7234-3684",
+    "email": "matt.buysr@datacite.org",
+    "image_url": "/images/mbuys.jpg",
```
Comment (kjgarza) [inline source/data/authors.json:63]: you photo is also missing from the repo. it might need to be added

**Comment by mjbuys** [comment]
@mfenner @kjgarza I will close this pull request as I am listed as an author in https://github.com/datacite/blog/blob/master/data/authors.json and do not need segugio.

**Comment by mfenner** [comment]
Authors are managed for a few months now in the blog: https://github.com/datacite/blog/blob/master/data/authors.json

**Pull request closed without merging**